com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueProcessor.cs

```diff
@@ -60,7 +61,17 @@ public void ProcessReceiveQueue(NetworkUpdateStage currentStage, bool isTesting)
 
                         if (!isTesting)
                         {
-                            m_NetworkManager.InvokeRpc(currentQueueItem);
+                            try
+                            {
+                                m_NetworkManager.InvokeRpc(currentQueueItem);
+                            }
+                            catch
+                            {
+                                if (NetworkLog.CurrentLogLevel <= LogLevel.Normal)
+                                {
+                                    NetworkLog.LogWarning($"A {currentQueueItem.QueueItemType} threw an exception and skipped its execution, please check Unity logs for more information.");
```

Comment:
This should be log error no? Our users are used to seeing red if something breaks. If we use warning many people will ignore this.

Reply:
yes but no :)
let me update the code again (I was still actively working on it)